File: auto-docs/executables/r/2d-histogram.r
library(plotly)

p <- plotly(username='theengineear', key='o9zlr0hy6z')

x <- rnorm(500)
y <- rnorm(500)+1
data <- list(
  list(
    x = x, 
    y = y, 
    type = "histogram2d"
  )
)

response <- p$plotly(data, kwargs=list(filename="2d-histogram", fileopt="overwrite", auto_open="FALSE"))
url <- response$url
filename <- response$filename

File: auto-docs/executables/node/2d_histogram.js
var plotly = require('plotly')('TestBot', 'r1neazxo9w')

var x = Math.random() * 50
var y = Math.random() * 50 + 1
var data = [
  {
    x: x, 
    y: y, 
    type: "histogram2d"
  }
];

var graph_options = {filename: "2d-histogram", fileopt: "overwrite", auto_open: "false"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});